superset/dashboards/schemas.py

```diff
@@ -169,7 +169,7 @@ class RolesSchema(Schema):
 class TagSchema(Schema):
     id = fields.Int()
     name = fields.String()
-    type = fields.String()
+    type = fields.Int()
```

Comment:
What changed here? Does this map to an enum? If so, can we use `fields.Enum` instead?